beginning.md

```diff
@@ -39,16 +40,19 @@ Anything written afer *#* will be ignored.
 ``` 
 
 ## Variables
-* Variables in Python allows us to store information and give it a label
-  we can use to retrieve that information later.
+* Variables in Python allows us to store information and give it a label.
+  We can use to retrieve that information later.
+
+* We can store numbers, strings (*a sequence of characters*) and complex data
+  types.
 
-* We can store numbers, strings (a squenece of characters) and complex data
-  types
 * We assign values to variables by putting the value to the right of an equal
   sign.
-* Since Python is a dynamic language so we donot need to define the data type of
+
+* Since Python is a [dynamic language](https://www.geeksforgeeks.org/why-python-is-called-dynamically-typed/) so we don't need to define the data type of
```

Comment:
It is a general purpose language, if we have idea in mind, we can easily prototype into project.